workflow/rules/diffexp.smk

```diff
@@ -142,6 +159,31 @@ rule tpm_matrix:
     script:
         "../scripts/sleuth-to-matrix.R"
 
+rule plot_group_density:
+    input:
+        "results/sleuth/all.rds"
+    output:
+        report("results/plots/group_density/{model}.group_density.pdf", caption="../report/group-density.rst", category="Densities")
```

Comment:
One last thing, sorry...

I would probably just stick those plots into the QC category for now, otherwise the report might get a bit unwieldy. If in the end this category gets overfilled, we'll have to think of a strategy for more meaningful categories.